Client<>AS-OAPI.yaml

```diff
@@ -0,0 +1,698 @@
+                interact_ref:
+                  type: string
+                  description: |-
+                    The interaction reference generated for this
+                    interaction by the AS.
+              required:
+                - interact_ref
+            examples:
+              Interaction Reference:
+                value:
+                  interact_ref: ad82597c-bbfa-4eb0-b72e-328e005b8689
+      description: Continue a grant request during or after user interaction.
+      tags:
+        - grant
+    delete:
+      summary: Cancel Grant
+      operationId: delete-continue
+      responses:
+        '202':
+          description: Accepted
+        '401':
+          description: Unauthorized
+        '404':
+          description: Not Found
+      description: Cancel a grant request or delete a grant client side.
+      tags:
+        - grant
+  '/token/{id}':
+    parameters:
+      - schema:
+          type: string
+        name: id
+        in: path
+        required: true
+    post:
+      summary: Rotate Access Token
+      operationId: post-token
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  access_token:
+                    $ref: '#/components/schemas/access_token'
+                required:
+                  - access_token
+              examples:
+                New access token:
+                  value:
+                    access_token:
+                      value: OZB8CDFONP219RP1LT0OS9M2PMHKUR64TB8N6BW7
+                      manage: 'https://openpayments.guide/auth/token/8f69de01-5bf9-4603-91ed-eeca101081f1'
+                      expires_in: 3600
+                      access:
+                        - type: outgoing-payment
+                          actions:
+                            - create
+                            - read
+                          identifier: 'https://openpayments.guide/alice'
+                          limits:
+                            interval: 'R12/2019-08-24T14:15:22Z/P1M'
+                            receiver: 'https://openpayments.guide/bob/incoming-payments/48884225-b393-4872-90de-1b737e2491c2'
+                            sendAmount:
+                              value: '500'
+                              assetCode: USD
+                              assetScale: 2
+        '401':
+          description: Unauthorized
+        '404':
+          description: Not Found
+      description: Management endpoint to rotate access token.
+      tags:
+        - token
+    delete:
+      summary: Revoke Access Token
+      operationId: delete-token
+      description: Management endpoint to revoke access token.
+      responses:
+        '204':
+          description: No Content
+        '401':
+          description: Unauthorized
+      tags:
+        - token
+components:
+  schemas:
```

Comment:
I think we should move the shared (or even all schemas) into a separate file (or individual files per schema?) and access them via relative remote references, such as:
```
$ref: 'schemas.json#/access
```
https://swagger.io/docs/specification/using-ref/

Reply:
maybe even `shared/schemas.json` if our current `open-api-spec.yaml` also ends up referencing it

Reply:
I wonder, however, if readme.io can deal with shared schemas 🤔